egs/tedlium/asr1/cmd.sh

```diff
@@ -11,10 +11,10 @@
 # or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
 
 export train_cmd="run.pl --mem 2G"
-export cuda_cmd="run.pl --mem 2G --gpu 1"
+export cuda_cmd="run.pl --mem 2G --gpu $num_gpu"
 export decode_cmd="run.pl --mem 4G"
```

Comment:
How about the overwrite in run.sh with `--gpuP option for `$cuda_cmd` ?
For example, `${cuda_cmd} --gpu ${num_gpu} ${expdir}/train.log`

Reply:
In this case we need to update all the egs examples?

Reply:
Yes, this is a good point, but just focusing on this recipe for now.